src/GeneralStateTestsFiller/stEWASMTests/createNonzeroFiller.yml

```diff
@@ -23,37 +52,39 @@ createNonzero:
     #       (import "ethereum" "getCallValue" (func $getCallValue (param i32)))
     #       (import "ethereum" "return" (func $return (param i32 i32)))
     #       (memory 1)
+    #       ;; simple contract that writes one value to storage, see source code above
+    #       (data (i32.const 384) "\00\61\73\6d\01\00\00\00\01\09\02\60\02\7f\7f\00\60\00\00\02\19\01\08\65\74\68\65\72\65\75\6d\0c\73\74\6f\72\61\67\65\53\74\6f\72\65\00\00\03\02\01\01\05\03\01\00\01\07\11\02\06\6d\65\6d\6f\72\79\02\00\04\6d\61\69\6e\00\01\0a\26\01\24\01\02\7f\41\80\05\21\00\41\a0\05\21\01\20\00\41\04\36\02\00\20\01\41\ef\fd\02\36\02\00\20\00\20\01\10\00\0b")
     #       (export "memory" (memory 0))
     #       (export "main" (func $main))
 
     #       (func $main
     #         ;; locals
     #         (local $memPointerCallValue i32)
-    #         (local $memPointerStorageKey3 i32)
+    #         (local $memPointerStorageKey4 i32)
     #         (local $memPointerReturnData i32)
 
     #         ;; memory layout
-    #         (set_local $memPointerCallValue (i32.const 256))
-    #         (set_local $memPointerStorageKey3 (i32.const 288))
-    #         (set_local $memPointerReturnData (i32.const 320))
+    #         (set_local $memPointerCallValue (i32.const 512))
+    #         (set_local $memPointerStorageKey4 (i32.const 544))
+    #         (set_local $memPointerReturnData (i32.const 384))
 
     #         ;; initialize
-    #         (i32.store (get_local $memPointerStorageKey3) (i32.const 2))
-    #         (i32.store (get_local $memPointerReturnData) (i32.const 0xabcd))
+    #         (i32.store (get_local $memPointerStorageKey4) (i32.const 3))
 
     #         ;; get call value
     #         ;; result gets stored in pointer loc
     #         (call $getCallValue (get_local $memPointerCallValue))
 
     #         ;; write it to storage
-    #         (call $storageStore (get_local $memPointerStorageKey3) (get_local $memPointerCallValue))
+    #         (call $storageStore (get_local $memPointerStorageKey4) (get_local $memPointerCallValue))
```

Comment:
Do we still need to call $getCallValue and store the result of  here?

Reply:
Why not? It's checked in the expect section on line 207. Strictly speaking value transfer can be checked by expecting a particular balance, but I don't see any harm in this check.

Reply:
OK I see it now, thanks Lane, no more comments